app/views/posts/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="container mt-5">
+  <!-- Sección de usuario -->
+  <div class="d-flex align-items-start mb-4">
+    <div class="p-3">
+      <img src="path_to_default_profile_pic.jpg" alt="Profile" class="img-thumbnail" style="width: 100px; height: 100px;">
+    </div>
+    <div class="border p-3 flex-grow-1 d-flex flex-column">
+      <h1><%= @user.name %></h1>
+      <div class="mt-auto text-right">
+        <small><%= @user.posts_counter %> posts</small>
+      </div>
+    </div>
+  </div>
+
+  <!-- Sección de lista de posts -->
+  <h2 class="mb-4">All Posts by <%= @user.name %></h2>
+  
+  <% @posts.each do |post| %>
+    <div class="card mb-4">
+      <div class="card-body">
+        <%= link_to post.title, user_post_path(@user, post) %>
+      </div>
+    </div>
+    
+    <!-- Comentarios para el post actual -->
+    <div class="card mt-2">
+      <div class="card-body">
+        <h5>Comments for <%= post.title %></h5>
+        <ul class="list-group">
+          <% post.comments.limit(5).each do |comment| %>
+              <li class="list-group-item"><%= comment.text %></li>
+          <% end %>
```

Comment:
- [ ] Please implement the wireframe, which includes comments and likes counter for a post. Also, show the username of each commenter as in the layout.
![image](https://github.com/EvansSnave/Blog_app_rails/assets/91486791/c4c6fefd-16f3-4850-bcd5-0c654559702c)
![image](https://github.com/EvansSnave/Blog_app_rails/assets/91486791/99f5f057-c0e3-4d9c-a449-707fe0281d02)